experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -77,10 +77,14 @@ import {
 import { OTLPLogExporter as OTLPProtoLogExporter } from '@opentelemetry/exporter-logs-otlp-proto';
 import { OTLPLogExporter as OTLPHttpLogExporter } from '@opentelemetry/exporter-logs-otlp-http';
 import { OTLPLogExporter as OTLPGrpcLogExporter } from '@opentelemetry/exporter-logs-otlp-grpc';
+import { OTLPTraceExporter as OTLPProtoTraceExporter } from '@opentelemetry/exporter-trace-otlp-proto';
+// import { OTLPTraceExporter as OTLPHttpTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
```

Comment:
is this not used on purpose? 🤔

Reply:
Good catch! My intention was to add a test to make sure the exporter is resolved from env. I'll add it in the next commit :)